src/bower.json

```diff
@@ -7,5 +7,10 @@
     "leaflet": "^0.7.3",
     "multiple-select": "~1.1.0",
     "ramda": "0.8.0"
+  },
+  "overrides": {
+    "leaflet": {
+      "ignore": true
```

Comment:
Maybe we need to modify the way bower images are copied over. We're currently sticking them all in one images directory, which sounds nice in theory, but pretty much all css uses relative image links and would require css overrides. We should probably try to maintain that directory structure when copying them over, or something along those lines.